src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px;
+}
+
+.search__form {
+  position: relative;
+}
+
+.search__input {
+  box-sizing: border-box;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  width: 100%;
+  outline: none;
+  height: 70px;
+  padding-left: 62px;
+  margin: 0 0 20px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__img {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+  pointer-events: none;
```

Comment:
why so? it is a part of input. when I click on it I want input to be focused. let's wrap pic in label tag to do so